Changelog.md

```diff
@@ -1,2254 +1,2328 @@
 ### 0.8.8 (unreleased)
 
 Language Features:
- * Inheritance: A function that overrides only a single interface function does not require the ``override`` specifier.
```

Comment:
This file should be untouched. Could you please remove the changes in the PR? The last commit "changelog updated" should be removed.

Reply:
Sorry. The Solidity documentation for contributing indicated that the changelog should be updated when making a PR; It has been removed.

Reply:
No worries. We only change it when user facing things need to be updated. In this case, `isoltest` is only an internal tool and therefore doesn't need a changelog entry.

Reply:
Yeah, looks like the [Contributing docs > Workflow for pull requests](https://docs.soliditylang.org/en/latest/contributing.html#workflow-for-pull-requests) is a bit vague about that. It does not say that it's only for user-facing stuff.

```
New features and bugfixes should be added to the `Changelog.md` file: please follow the style of previous entries, when applicable.
```

We should clarify it.